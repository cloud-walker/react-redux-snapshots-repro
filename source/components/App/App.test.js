import React from 'react'
import {configure, shallow} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import {App} from './'

configure({adapter: new Adapter()})

test('App', () => {
  const subject = shallow(<App />)

  expect(subject).toMatchSnapshot()
})
