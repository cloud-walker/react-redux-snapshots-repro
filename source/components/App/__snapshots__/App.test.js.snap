// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App 1`] = `
<div>
  <h1>
    Funky App
  </h1>
  <Dummy />
</div>
`;
